configuration.k

```diff
@@ -0,0 +1,228 @@
+
+module CONFIGURATION
+
+imports SOLIDITY-SYNTAX
+
+
+syntax START
+
+
+configuration
+    <k> $PGM:START </k>
+    <exit-code exit=""> 1 </exit-code>
+    // ... 
+    
+    <ethereum>
+        
+        // Solidity execution engine
+        // =========================
+        
+        <solidity> 
+            
+            // Mutable during a single transaction
+            // -----------------------------------
+            
+            <execEngine>
+                <output>          .K  </output>
+                <statusCode>      .K  </statusCode>
+                <callStack>     .List  </callStack>
+                <interimStates>   .K  </interimStates>
+                <touchedAccounts> .K  </touchedAccounts>
```

Comment:
Why are `<callStack>` and `<interimStates>` starting with `.K` instead of `.List`? In the semantics you use them like they are `List`.